.github/workflows/test-gpu.yml

```diff
@@ -51,19 +51,21 @@ jobs:
       - name: Nvidia SMI sanity check
         run: nvidia-smi
 
-      - uses: mamba-org/setup-micromamba@v1
+      - name: Install Python
+        uses: actions/setup-python@v5
         with:
-          micromamba-version: "1.5.6-0"
-          environment-file: ci/gpu_ci.yml
-          post-cleanup: "all"
+          python-version: "3.x"
```

Comment:
Probably want to fix this to a version?

Reply:
Or multiple?

Reply:
I think this uses the newest stable release thats what we do for the release aswell

Reply:
In the release workflow, we don’t actually import the package AFAIK, so it’s fine if we build it with a version that isn’t supported by e.g. numba yet.

I agree with the sentiment: I don’t want multiple workflows that all hardcode the newest release that numba (and by extension we) support, I’d like to outsource or at least centralize this release specifier.

It’s not super important right now though, as we’ll notice when the next stable release breaks numba …